docassemble/ALDashboard/data/questions/nav.yml

```diff
@@ -1,4 +1,11 @@
 ---
+metadata:
+  required privileges:
+    - admin
+    - developer
+  temporary session: True
+  sessions are unique: True
```

Comment:
Do we want these two on everything? I forget exactly how they interact with stuff

Reply:
`temporary session` means its discarded and you always start fresh. `sessions are unique` means that you never have more than one of each session at a time.

Given these are all single-interaction quick utilities, yes, they should be safe for all of the dashboard interviews. Prevents it from cluttering up your saved answer list and makes sure you always start fresh.

Reply:
gotcha, sounds good.